openapi.yaml

```diff
@@ -477,6 +514,15 @@ paths:
 
             - noFileAccess: The user is not registered as a Data Submitter for the
             corresponding file.'
+        '404':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HttpUnknownStorageAliasError'
```

Comment:
Because of the above, the code for this error should be 400.

Reply:
Probably also needs an additional test, I'm on it